samples/django/README.md

```diff
@@ -0,0 +1,17 @@
+# Django Samples
+
+These two sample Django apps provide a skeleton for the two main use cases of the
+Django contrib helpers
```

Comment:
"contrib" isn't an English word but it is a term of art we use in our code. Consider putting it in ticks so that it appears as `contrib`?